lib/utils/slugify.js

```diff
@@ -0,0 +1,9 @@
+import slugify from 'slugify'
```

Comment:
please use this function instead: https://github.com/orbiting/publikator-frontend/blob/master/lib/utils/slug.js (it's even tested)

(move to styleguide or copy & paste)

Reply:
keeping dependencies at bay ;)